web_src/css/base.css

```
@@ -17,6 +17,8 @@
   --octicon-chevron-right: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16" width="16" height="16"><path d="M6.22 3.22a.75.75 0 0 1 1.06 0l4.25 4.25a.75.75 0 0 1 0 1.06l-4.25 4.25a.751.751 0 0 1-1.042-.018.751.751 0 0 1-.018-1.042L9.94 8 6.22 4.28a.75.75 0 0 1 0-1.06Z"></path></svg>');
   /* non-color variables */
   --border-radius: 0.28571429rem;
```

```suggestion
  --border-radius: 4px;
```

With 14px base font size, this comes out as `3,9998px`, so let's make it integer.

---

That's simply what we got from Fomantic, so I mean we CAN change it but we don't need to…